serverless-http.d.ts

```diff
@@ -13,10 +13,10 @@ declare namespace ServerlessHttp {
     }
     type HandlerCompatibleApp = Function | Partial<FrameworkApplication>;
     type LambdaPartial = (
-        event: any,
+        event: AWSLambda.APIGatewayProxyEvent,
         context: AWSLambda.Context,
-        callback?: AWSLambda.Callback
-    ) => void;
+        callback?: AWSLambda.APIGatewayProxyCallback
```

Comment:
It doesn't take callback anymore is it worth removing this?

Reply:
Sure, I will remove this